reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -267,6 +272,14 @@ class FreemarkerTemplateProcessor(
             licenses.groupBy { it.license }
                 .map { (_, licenses) -> licenses.merge() }
                 .sortedBy { it.license.toString() }
+
+        /**
+         * Return `true` if there are any unresolved issues, or `false` otherwise.
+         */
+        @Suppress("UNUSED") // This function is used in the templates.
+        fun hasUnresolvedIssues() = ortResult.collectIssues().values.flatten().any { issue ->
+            resolutionProvider.getIssueResolutionsFor(issue).isEmpty()
+        }
```

Comment:
Formatting should be:

        fun hasUnresolvedIssues() =
            ortResult.collectIssues().values.flatten().any { issue ->
                resolutionProvider.getIssueResolutionsFor(issue).isEmpty()
            }